cartographer/mapping/2d/proto/submaps_options_2d.proto

```diff
@@ -14,18 +14,17 @@
 
 syntax = "proto3";
 
-import "cartographer/mapping/2d/proto/range_data_inserter_options_2d.proto";
+import "cartographer/mapping/2d/proto/grid_2d_options.proto";
+import "cartographer/mapping/2d/proto/probability_grid_range_data_inserter_options_2d.proto";
+import "cartographer/mapping/proto/range_data_inserter_options.proto";
 
 package cartographer.mapping.proto;
 
 message SubmapsOptions2D {
-  // Resolution of the map in meters.
-  double resolution = 1;
-
   // Number of range data before adding a new submap. Each submap will get twice
   // the number of range data inserted: First for initialization without being
   // matched against, then while being matched.
-  int32 num_range_data = 3;
-
-  RangeDataInserterOptions2D range_data_inserter_options = 5;
+  int32 num_range_data = 1;
+  GridOptions2D grid_options_2d = 2;
```

Comment:
does this need to be introduced (i.e., will you have more options in it in the future)? it will break old pbstreams

Reply:
Same as above.